coderd/prometheusmetrics/aggregator.go

```diff
@@ -216,15 +332,41 @@ func (ma *MetricsAggregator) Run(ctx context.Context) func() {
 			case outputCh := <-ma.collectCh:
 				ma.log.Debug(ctx, "collect metrics")
 
+				var input []annotatedMetric
 				output := make([]prometheus.Metric, 0, len(ma.store))
-				for _, m := range ma.store {
+
+				// If custom aggregation labels have not been chosen, generate Prometheus metrics without any pre-aggregation.
+				// This results in higher cardinality, but may be desirable in larger deployments.
+				// Default behavior.
```

Comment:
nit: maybe add an empty line before this comment. Otherwise, it looks like a single paragraph.